src/components/Toolinformation.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Button from './Button'
+import logoBackground from './images/Background.png';
+
+const Toolinformation = ({addbtn, toolname}) => {
+  return (
+    <div className='toolInformation'>
+        <img src={logoBackground}></img>
+        <h1 className='toolInformationText1'>Overzicht</h1>
```

Comment:
If you like to follow the HTML semantics strictly, having two Heading 1 tags within the same scope is not allowed. I don't think it will be an issue for now. It's sometimes also not possible (at least not easily).